.travis.yml

```diff
@@ -76,7 +76,7 @@ before_script:
   fi
 - npm --version
 - node --version
-- nvm install 10.13.0
+- nvm install --latest-npm
```

Comment:
nvm install nodejs and you removed it to update npm. I think it should stay as before (or use newer node.. don't know the requirements.

`nvm install --latest-npm` tries to update npm

Reply:
Hm, isn't this what Gutenberg does?

Reply:
https://github.com/WordPress/gutenberg/blob/4cc695898cedc8eb97b2bc2e4c63217cd17e6244/.travis.yml#L40

Reply:
I'm looking into Travis CI, as I haven't used it before, but it's definitely a different command, that does the above. Maybe node is already installed or installed in some other place.

Since it was there in WordPress Travis, it should stay there, except some change in `node` installation has happened.

Also the command `--latest-npm` is in different Travis hooks:
Wordpress: `before_script`
GB: `before_install`

Reply:
With nodejs installation comes the npm too.. So it might explain the `Puppeteer not found` module... ?

Reply:
![Screenshot(1)](https://user-images.githubusercontent.com/16275880/85842015-a2626400-b7a7-11ea-9a1d-41c418d075ab.png)

So it actually installs new nodejs

Reply:
If Puppeteer isn’t installed then it means that the lock file wasn’t properly updated. The simplest way to fix it is to install the alias that  `wp-scripts` uses as a dev dependency for the project, example: 


https://github.com/WordPress/gutenberg/blob/7060b11b9b9efc00e099c9092fe520402d77ee4b/package.json#L180

It’s a bug in npm’s implementation.

Reply:
We're still getting this error:

```
Downloading Chromium r737027 - 118.4 Mb [====================] 100% 0.0s 
Chromium (737027) downloaded to /Applications/MAMP/htdocs/wordpress-git/node_modules/@wordpress/scripts/node_modules/puppeteer/.local-chromium/mac-737027
Error: Could not find browser revision 737027. Run "npm install" or "yarn install" to download a browser binary.
```

Reply:
Looks like Chromium is downloaded to `node_modules/@wordpress/scripts/node_modules/puppeteer`, while puppeteer runs from `node_modules/puppeteer`.